Configurations/10-main.conf

```diff
@@ -943,13 +943,24 @@ my %targets = (
         multilib         => "64",
     },
 
+    #### Alpha Linux
+    # linux-alpha+bwx-gcc is meant to be used from ./config only. If you run
+    # './Configure linux-alpha+bwx-gcc' manually, do complement the command
+    # line with -mcpu=ev56, -mcpu=ev6 or whatever is appropriate.
     "linux-alpha-gcc" => {
         inherit_from     => [ "linux-generic64" ],
         lib_cppflags     => add("-DL_ENDIAN"),
         bn_ops           => "SIXTY_FOUR_BIT_LONG",
         asm_arch         => 'alpha',
         perlasm_scheme   => "void",
     },
+    "linux-alpha+bwx-gcc" => {
+        inherit_from     => [ "linux-generic64" ],
+        lib_cppflags     => add("-DL_ENDIAN"),
+        bn_ops           => "SIXTY_FOUR_BIT_LONG RC4_CHAR",
```

Comment:
This appears to be the same bn_ops as inherited from liunx-generic64, so this line seems redundant

Reply:
Thanks, fixed.